pkg/schedule/coordinator.go

```diff
@@ -481,6 +499,8 @@ func (c *Coordinator) InitSchedulers(needRun bool) {
 	if err := c.cluster.GetSchedulerConfig().Persist(c.cluster.GetStorage()); err != nil {
 		log.Error("cannot persist schedule config", errs.ZapError(err))
 	}
+
+	c.markSchedulersInitialized()
```

Comment:
Is it necessary for PD mode?

Reply:
It's only used in the scheduling service, but I think leaving the mark here is okay, which is not an expensive operation.